test_reporting/sai_coverage/case_scanner_runtime.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import json
+import os
+import re
+import uuid
+
+
+def get_parser(description="Runtime Scanner"):
+    """
+    Parse command line
+    """
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument("--log_path", "-l", type=str,
+                        default="log", help="log path to scan.")
+    parser.add_argument("--info_file", "-i", type=str,
+                        default="log", help="test platform info file path to scan.")
+    parser.add_argument("--result_path", "-r", type=str, default="result",
+                        help="path of the parsed result.")
+    args = parser.parse_args()
+    return args
+
+
+def get_test_platform(path):
+    with open(path, 'r') as f:
+        for line in f:
+            if 'Platform:' in line:
+                return line.split()[1]
+
+
+def parse_log(log_path, result_path, test_platform):
+    results = []
```

Comment:
please add comments for each method
please add sample logs in the parse_log method
and add some log sections during parsing logs